docs/en/sql-reference/functions/conditional-functions.md

```diff
@@ -22,8 +22,8 @@ You can use the [short_circuit_function_evaluation](../../operations/settings/se
 **Arguments**
 
 -   `cond` – The condition for evaluation that can be zero or not. The type is UInt8, Nullable(UInt8) or NULL.
--   `then` – The expression to return if condition is met.
--   `else` – The expression to return if condition is not met.
+-   `then` – The expression to return if condition is true.
```

Comment:
While you are at it, could you please also change "evaluate" to "evaluates" in l. 10? Thanks!

Reply:
The previous formulation "met" was not wrong or imprecise. Anyways, we can change it to "true" (l. 25)  / "false" (l. 26).